Lesson4.ipynb

```diff
@@ -29,6 +32,963 @@
     "\n",
     "Select **Lesson4.ipynb**"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Read CSV file"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### What is CSV file?\n",
+    "\n",
+    "A CSV is a comma separated values file which allows data to be saved in a table structured format. A CSV file is similar to an Excel spreadsheet, though it doesn't have the style formatting and has a .csv extension instead. Traditionally they take the form of a text file containing information separated by commas, hence the name.\n",
+    "\n",
+    "** Content in CSV Sample **\n",
+    "\n",
+    "Start from this lesson, we will use the data from Biggest Loser. For those unfamiliar with the Biggest Loser step challenge, this was a 4-week long challenge where teams (each consisting 5 members) combined their daily step count in order to reach these goals:\n",
+    "\n",
+    "    1. to be the first team to reach 1 million steps\n",
+    "\n",
+    "    2. to have the highest overall step count amongst all teams\n",
+    "\n",
+    "    3. to show the most improved team step count (from weeks 1+2 to weeks 3+4)\n",
+    "\n",
+    "*** The following sample data is from the Biggest Loser step challenge. ***\n",
+    "\n",
+    "     team_no,team_name,team_captain,team_member,2018-04-02,2018-04-03\n",
+    "\n",
+    "     1,TBD,FALSE,1-1,11980,10437\n",
+    "\n",
+    "     1,TBD,FALSE,1-2,22935,13399\n",
+    "\n",
+    "However, CSV file is a table, as we are familiar with Excel, to make it more clear, we can have a comparison with Excel we generally use in our daily work. The first row is the coloumn name, start from the second row, they are the specific information of each person.\n",
+    "\n",
+    "If you open the CSV file using a text Editor such as Notepad, you'll see the content as above.\n",
+    "\n",
+    "If you open the CSV file using Excel, the content will have the same format as a .xls file."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## How to read CSV file in Python? ##\n",
+    "\n",
+    "To read and analyse the data, we use the Python csv library. CSV literally stands for comma separated variable, where the comma is what is known as a \"delimiter.\"\n",
+    "\n",
+    "    Tips: Whatever the library you need, you should import all the necessary libraries at the beginnng of the code.\n",
+    "    \n",
+    "## Let's start\n",
+    "\n",
+    "### Print the csv file\n",
+    "\n",
+    "    1. The first thing we need to do is importing CSV, then we can use the functions under Python csv library.\n",
```

Comment:
"The first thing we need to do is importing CSV, then we can use the functions under Python csv library." should read "The first thing we need to do is import the Python CSV library, so we can use the functions in that library."